x-pack/plugins/enterprise_search/public/applications/shared/layout/layout.tsx

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { EuiPage, EuiPageSideBar, EuiPageBody } from '@elastic/eui';
+
+import './layout.scss';
+
+export const Layout: React.FC = ({ children }) => {
+  return (
+    <EuiPage className="enterpriseSearchLayout">
+      <EuiPageSideBar className="enterpriseSearchLayout__sideBar" />
+      <EuiPageBody className="enterpriseSearchLayout__body">{children}</EuiPageBody>
+    </EuiPage>
+  );
+};
```

Comment:
```suggestion
export const Layout: React.FC = ({ children }) => (
  <EuiPage className="enterpriseSearchLayout">
    <EuiPageSideBar className="enterpriseSearchLayout__sideBar" />
    <EuiPageBody className="enterpriseSearchLayout__body">{children}</EuiPageBody>
  </EuiPage>
);
```
Small nit: do we need to explicitly return?

Reply:
The only (very minor) reason I've been preferring `=> { return () }` over `=> ()` recently is for indenting and git diffing. `{ return () ` allows you to add vars, context, state, etc. without changing the indenting of the returned jsx and generating a more annoying diff blob 🤷 

I'm poking at how responsive behavior might work in this PR and it's _possible_ I might have to add a state in this file for tracking the open/closed toggling of the nav on mobile, so will keep an eye on this - if I don't, then I can change it to `=> ()`

Reply:
Resolving: the Layout component is now managing nav mobile UX state (open/closed), so needs the current syntax/indenting